src/PcmPlayer.ts

```diff
@@ -40,9 +40,9 @@ export class PcmPlayer {
     this.feedWorklet(source)
   }
 
-  volume(volume: number) {
+  volume(volume: number, duration: number =0) {
     if (this.gainNode) {
-      this.gainNode.gain.value = volume
+      this.gainNode.gain.setTargetAtTime(volume, this.context!.currentTime + duration, duration / 3)
```

Comment:
Changed this to use set target time, so the volume adjust is not instant. The time constant is a third of the duration, which results in 95% volume at the end of the duration

https://developer.mozilla.org/en-US/docs/Web/API/AudioParam/setTargetAtTime#choosing_a_good_timeconstant